src/cpp/rtps/builtin/discovery/participant/PDP.cpp

```diff
@@ -818,6 +827,7 @@ WriterProxyData* PDP::addWriterProxyData(
                 WriterDiscoveryInfo info(*ret_val);
                 info.status = WriterDiscoveryInfo::DISCOVERED_WRITER;
                 listener->onWriterDiscovery(mp_RTPSParticipant->getUserRTPSParticipant(), std::move(info));
+                check_and_notify_type_discovery(listener, *ret_val);
```

Comment:
Again, this kind of changes have been reviewed by @richiprosima , right?

Reply:
Yes.